src/Explorer/Controls/NotebookGallery/CodeOfConductComponent.tsx

```diff
@@ -68,7 +68,7 @@ export class CodeOfConductComponent extends React.Component<CodeOfConductCompone
   public render(): JSX.Element {
     if (!this.viewCodeOfConductTraced) {
       this.viewCodeOfConductTraced = true;
-      trace(Action.NotebooksGalleryViewCodeOfConduct);
+      trace(Action.NotebooksGalleryViewCodeOfConduct, ActionModifiers.Mark, {});
```

Comment:
Should we just fix trace method to always pass {} instead of undefined? Not sure if we’ll need to fix other usage of this method also.

Reply:
changed it